Makefile

```diff
@@ -3,6 +3,8 @@ VERSION ?= 2.0.2
 # Default bundle image tag
 IMAGE_TAG_BASE ?= controller
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+# Makefile uses sh shell by default but we want to use bash
+SHELL := /bin/bash
```

Comment:
Why?

Reply:
How did it work up until now?

Reply:
I think on our local systems the env var SHELL is already set to bash shell. But virtual machine provided by github actions don't have SHELL env var set so by default Makefile uses sh shell. We can solve the issue by either setting it via Makefile or having a job in github actions that sets SHELL env var.
I think we should set it via Makefile because it will help to run test without issue if someone is using any other shell that doesn't support the source.

Reply:
https://www.gnu.org/software/make/manual/make.html#Choosing-the-Shell

Reply:
ok